plugins/xlsx-extractor/src/utils.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { prependNonUniqueHeaderColumns } from './utils'
+
+describe('utils', () => {
+  describe('prependNonUniqueHeaderColumns', () => {
```

Comment:
Style feedback, so take it or leave it, but adding new lines between your setup and verify phases would go a long way for readability: https://thoughtbot.com/blog/four-phase-test

```typescript
it('should handle non-unique values correctly', () => {
  const input = { a: 'value', b: 'value', c: 'value' }
  const expected = { a: 'value', b: 'value_1', c: 'value_2' }
  
  expect(prependNonUniqueHeaderColumns(input)).toEqual(expected)
})
```

Reply:
Thank you. It was a copy and paste from ChatGPT. lol. Ill update it.